### PR TITLE
My portofolio challenge1

### PR DESCRIPTION
**Pull request by sarazaToussaint**
In this Portfolio challenge1 we: 

- Added the navbar for the homepage
- Added the headline section
- Configurated linter for HTML/CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    <title>Portofolio</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+    <main>
+        <nav>
+            <h3>My Logo</h3>
+            <img src="./images/list.svg" alt="" />
+        </nav>
```
Comment (karvel-code) [inline index.html:19]: - [ ] This section should be wrapped inside the `<header>` tag inorder to improve on the semantics of your code.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300&display=swap" rel="stylesheet" />
+
+    <title>Portofolio</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+    <main>
+        <nav>
+            <h3>My Logo</h3>
```
Comment (karvel-code) [inline index.html:17]: - [ ] The logo should be a link, ie should be wrapped inside the `<a>` tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            <img src="./images/list.svg" alt="" />
```
Comment (karvel-code) [inline index.html:18]: - [ ] Wrap only the hamburger icon inside the `<nav>` tag for semantic reasons.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+            </div>
+            <div class="links">
+                <a href="#">LET'S CONNECT</a>
+                <div class="link-y">
+                    <a href="#"><img src="./images/Vector4.svg" alt="Vector4"></a>
+                    <a href="#"><img src="./images/Vector6.svg" alt="Vector6"></a>
+                    <a href="#"><img src="./images/Vector5.svg" alt="Vector5"></a>
+                    <a href="#"><img src="./images/Vector1.png" alt="Vector1"></a>
+                    <a href="#"><img src="./images/Vector.png" alt="Vector"></a>
+                    <a href="#"><img src="./images/Vector3.svg" alt="Vector3"></a>
```
Comment (karvel-code) [inline index.html:40]: - [ ] Please wrap the icons inside the `<ul>` and `<li>` tag so as to improve on semantics.

**Review by karvel-code — CHANGES_REQUESTED** [review]
Hi @sarazaToussaint , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- [ ] You have adhered to git-flow 💯 
- [ ] All your linter checks are passing 👍 


## Required Changes ♻️
_Check the comments under the review._

- [ ] Please remove sections of your readme file that do not apply for this project or are blank. This will enhance your project's professionalism.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mre55 — APPROVED** [review]
**Hi @sarazaToussaint,**

# Status: APPROVED  🎉 🎊 

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 

Alt Text

Cheers and Happy coding!👏👏👏

## Optional suggestions 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

#### _I left some [OPTIONAL] suggestions make sure to check in the comment below_ 👇.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**